packages/object-deep-omit/index.cjs

```diff
@@ -0,0 +1,44 @@
+module.exports = deepOmit;
+
+/*
+let obj = {a:3, b:{c:5, d:7}, e:[1,2,3]};
+omit(obj, 'b.c'); // {a:3, b:{d:7}, e:[1,2,3]};
+omit(obj, ['a','b','c']); // {a:3, b:{d:7}, e:[1,2,3]};
+omit(obj, 'e.1'); // {a: 3, b:{c:5, d:7}, e:[1,3]};
+*/
+
+function deepOmit(obj, propsArg) {
```

Comment:
This util mutates in place and returns boolean
Original justOmit returns a mutated copy and preserves the original. I think we should be consistent with that